privly.safariextension/Info.plist

```diff
@@ -24,7 +24,7 @@
 				<key>Identifier</key>
 				<string>new_message</string>
 				<key>Title</key>
-				<string>New Message</string>
+				<string>Privly New Message</string>
```

Comment:
"Privly" serves as a replacement for the icon, so it should not be read as a sentence. It should be something like `Privly: New Message`.